src/Command/ViewCourseCommand.php

```diff
@@ -3,26 +3,28 @@
 namespace App\Command;
 
 use App\Command\Command;
+use App\Entity\User;
+use App\Entity\Course;
 
 class ViewCourseCommand implements Command
 {
     /**
-     * @var int 
+     * @var User
      */
-    public $userId;
-    
+    public $user;
```

Comment:
These attributes should be private and should be accessed by getters. This will make your command read only.